lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-redhat-infra_manager.rb

```diff
@@ -2,5 +2,6 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Redhat_InfraManager < MiqAeServiceEmsInfra
     expose :validate_import_vm
     expose :submit_import_vm
+    expose :submit_configure_imported_vm_networks
```

Comment:
@matobet Is there a reason behind the method naming being used here?  The name reads very strange to me.

Is `submit` required at all?  And will this only configure networks on imported VMs?  If not it feels like this is just `configure_vm_networks`.  Or if this logic was moved to the VM model it could be `configure_networks`.